xbmc/addons/interfaces/AddonInterfaces.h

```diff
@@ -51,12 +51,12 @@ namespace ADDON
     \*/
     static void*        AddOnLib_RegisterMe            (void* addonData);
     static void         AddOnLib_UnRegisterMe          (void* addonData, void* cbTable);
-    void*               AddOnLib_GetHelper()          { return m_helperAddOn; }
+    void*               AddOnLib_GetHelper() const    { return m_helperAddOn; }
     /*\_________________________________________________________________________
     \*/
     static void*        GUILib_RegisterMe              (void* addonData);
     static void         GUILib_UnRegisterMe            (void* addonData, void* cbTable);
-    void*               GUILib_GetHelper()            { return m_helperGUI; }
+    void*               GUILib_GetHelper() const      { return m_helperGUI; }
```

Comment:
does this change binary addons ABI?

Reply:
Yes it does.
Reference: https://community.kde.org/Policies/Binary_Compatibility_Issues_With_C%2B%2B#The_Do.27s_and_Don.27ts